ex3/Premier league.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<p>To get the number of unique items in a series, use <code>.nunique()</code>:</p><p>For example:</p><p> <code>df[df["Season"] == season]["HomeTeam"].nunique()</code></p><p> </p><p>Here, there is a very simple solution - try to count the number of games in each season (how can you do it by calling a single function ?). Why does it answer the question ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>